ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/AwsConnectorExecutor.java

```diff
@@ -126,6 +184,78 @@ public void invokeRemoteService(
         }
     }
 
+    @Override
+    public void invokeRemoteServiceStream(
+        String action,
+        MLInput mlInput,
+        Map<String, String> parameters,
+        String payload,
+        ExecutionContext executionContext,
+        ActionListener<Tuple<Integer, ModelTensors>> actionListener
+    ) {
+        try {
+            RemoteModelStreamProducer streamProducer = new RemoteModelStreamProducer();
+            StreamTicket streamTicket = streamManager.registerStream(streamProducer, null);
+            getLogger().debug("Stream ticket: {}", streamTicket);
+            List<ModelTensor> modelTensors = new ArrayList<>();
+            modelTensors.add(ModelTensor.builder().name("response").dataAsMap(Map.of("stream_ticket", streamTicket)).build());
+            threadPool.executor("opensearch_ml_predict_stream").execute(() -> {
+                actionListener.onResponse(new Tuple<>(0, new ModelTensors(modelTensors)));
+            });
+            String llmInterface = parameters.get(LLM_INTERFACE);
+            llmInterface = llmInterface.trim().toLowerCase(Locale.ROOT);
+            llmInterface = StringEscapeUtils.unescapeJava(llmInterface);
+            validateLLMInterface(llmInterface);
+
+            ConverseStreamRequest request = ConverseStreamRequest
```

Comment:
So this AWS connector executor only support Bedrock converse stream ? Suggest add  some java-doc/comments